serverless.yml

```diff
@@ -2,7 +2,7 @@ service: bitmovin-thumbnail-image-lambda
 
 provider:
   name: aws
-  runtime: nodejs4.3
+  runtime: nodejs8.10
```

Comment:
i see that node 8.10 will only be supported until dec 2019. why not upgrade to the most current version (10)

https://github.com/nodejs/Release#release-schedule

Reply:
ah ok